Modules/Filtering/ImageGrid/wrapping/itkShrinkImageFilter.wrap

```diff
@@ -1,3 +1,8 @@
 itk_wrap_class("itk::ShrinkImageFilter" POINTER)
+  itk_wrap_image_filter("${WRAP_ITK_RGB}" 2)
+  itk_wrap_image_filter("${WRAP_ITK_RGBA}" 2)
+# VS9 (2008) does not correctly implicitly cast itk::Vector for operator +=
```

Comment:
ITKv5 supports only VS2015 and newer. Workarounds for older version should be cleaned up when we come across them.

Reply:
In each of the two `.wrap` files, I removed the text comment and uncommented the subsequent line of code.